woocommerce-services.php

```diff
@@ -841,6 +848,12 @@ public function rest_api_init() {
 			$this->set_rest_self_help_controller( $rest_self_help_controller );
 			$rest_self_help_controller->register_routes();
 
+			require_once( plugin_basename( 'classes/class-wc-rest-connect-service-data-refresh-controller.php' ) );
```

Comment:
```suggestion
			require_once __DIR__ . '/classes/class-wc-rest-connect-service-data-refresh-controller.php';
```
Improved CS pattern introduced with https://github.com/Automattic/woocommerce-services/pull/2322